machines/migrations/0005_auto_20221123_0802.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.16 on 2022-11-23 08:02
```

Comment:
To be squased ?

Reply:
fixed: [638e381](https://github.com/fablab-chaux-de-fonds/Interlab/pull/137/commits/638e3812accae8a710693aae0b6aa1b692a96791)